sympy/plotting/plot.py

```diff
@@ -181,7 +184,10 @@ def __init__(self, *args, **kwargs):
         # The backend type. On every show() a new backend instance is created
         # in self._backend which is tightly coupled to the Plot instance
         # (thanks to the parent attribute of the backend).
-        self.backend = DefaultBackend
+        if 'usebackend' in kwargs:
```

Comment:
ref [this}(https://github.com/sympy/sympy/blob/master/sympy/plotting/plot.py#L184) doesn't self.backend mean 'backend' is an attribute of `Plot` class?